pkg/machinepool/machinepool.go

```diff
@@ -228,7 +228,7 @@ func (m *machinePool) CreateMachinePool(r *rosa.Runtime, cmd *cobra.Command, clu
 		}
 	}
 
-	minReplicas, maxReplicas, replicas, autoscaling, err := manageReplicas(cmd, args, multiAZMachinePool)
+	minReplicas, maxReplicas, replicas, autoscaling, err := manageReplicas(cmd, args, multiAZMachinePool, false)
```

Comment:
NP instead of false, would it be better to supply the boolean value from the cluster instead?
cluster.Hypershift().Enabled()